module/VuFind/src/VuFind/Config/PluginFactory.php

```diff
@@ -65,17 +65,23 @@ public function __construct()
     /**
      * Load the specified configuration file.
      *
-     * @param string $filename config file name
-     * @param string $path     path relative to VuFind base (optional; defaults
-     * to config/vufind
+     * @param string   $filename     Config file name
+     * @param ?string  $path         Path relative to VuFind base (optional; use null
+     * for default)
+     * @param callable $pathCallback Callback for getting an absolute config
+     * file path (optional; defaults to Locator::getConfigPath)
      *
      * @return Config
      */
-    protected function loadConfigFile($filename, $path = 'config/vufind')
-    {
+    protected function loadConfigFile(
+        $filename,
+        $path = null,
+        callable $pathCallback = null
```

Comment:
Would it make sense to make this non-optional, since it's an internal protected method, and we know we'll have access to the PathResolver? Then we'd have one less ongoing reference to the deprecated Locator...

Reply:
I ended up changing it to just take the full filename, which seemed much more intuitive.